app/models/admin_ability.rb

```diff
@@ -118,7 +118,10 @@ def signed_in_with_organizer_role(user, conf_ids_for_organization_admin = [])
     can :manage, Commercial, commercialable_type: 'Conference',
                              commercialable_id: conf_ids
     can :manage, Registration, conference_id: conf_ids
-    can :manage, RegistrationPeriod, conference_id: conf_ids
+    can :manage, RegistrationPeriod do |registration_period|
+      conference = registration_period.conference
+      conf_ids.include?(conference.id) && conference.tickets.for_registration.any?
```

Comment:
@Ana06 
It will also fail while accessing the `new_admin_conference_registration_period_path` because organizer of the conference will only have the `new` ability (Included in `manage`) for `registration period` when conference has at least one registration ticket. So if there are no registration tickets the  organizer cannot access `new_admin_conference_registration_period_path` as the organizer will not be authorized to access that page.